src/main/webapp/webpack/i18nThymeleafWebpackPlugin.js

```diff
@@ -10,7 +10,10 @@
 
 "use strict";
```

Comment:
Can you please beef up the comments in this file as well?  Some internal comments on some of the complicated methods would help, plus many of the functions just have param and return comments.  Please make a method level comment about what it actually does.  Example:
```js
/**
 * @param {string} request the path to the js file being requested
 * @returns {boolean} request is valid and is local
 */
```

Reply:
Updated in ba61b6d